android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -739,6 +739,16 @@ public void showPositionChooserForEditor(boolean isBusiness, boolean applyPositi
   private void showPositionChooser(ChoosePositionMode mode, boolean isBusiness, boolean applyPosition)
   {
     closeFloatingToolbarsAndPanels(false);
+    if (mMapFragment != null)
+    {
+      final View mapView = mMapFragment.getView();
+      if (mapView != null)
+      {
+      int width = mapView.getWidth();
+      int height = mapView.getHeight();
+      Framework.nativeSetVisibleRect(0, 0, width, height);
+      }
+    }
```

Comment:
ATM it is a bit strange "adjusting" the viewport in ```showPositionChooser``` function, no?

Reply:
Ok, I've got the point that we select a chooser with the opened PP and it forces PP to close.
Please, move forward with rebase.